package.json

```diff
@@ -23,14 +23,14 @@
     "lint": "eslint . --ext .js,.jsx",
     "jest:update": "jest --updateSnapshot",
     "jest:watch": "jest --watch",
-    "flow": "flow",
-    "flow-check-ios": "flow check",
-    "flow-check-android": "flow check --flowconfig-name .flowconfig.android",
+    "flow": "npm run flow-check-ios && npm run flow-check-android",
+    "flow-check-ios": "flow check --show-all-errors --show-all-branches",
+    "flow-check-android": "flow check --show-all-errors --show-all-branches --flowconfig-name .flowconfig.android",
     "check-bpk-dependencies": "",
     "fix-bpk-dependencies": "",
     "test": "npm run lint && npm run check-bpk-dependencies && jest --coverage && npm run flow-check-ios && npm run flow-check-android && npm run spellcheck",
     "check-owners": "npm whoami && ( node scripts/npm/get-owners.js | grep -E $(npm whoami) ) && node scripts/npm/check-owners.js",
-    "release": "npm run check-owners && git fetch && git pull --rebase && npm run flow stop && npm test && npm run android:test && npm run build && node scripts/npm/release/index.js",
+    "release": "npm run check-owners && git fetch && git pull --rebase && npm run flow && npm test && npm run android:test && npm run build && node scripts/npm/release/index.js",
```

Comment:
`npm run flow stop` was here to ensure that we don't have duplicated servers running locally - as it can break things.
`npm test` runs `flow` so no need to run here

Reply:
Good catch, unsure why I had changed that onw!